unitTests/ui/ResponsiveComponents/ResponsiveTabsTest.ts

```diff
@@ -106,6 +106,17 @@ export function ResponsiveTabsTest() {
         });
       };
 
+      const simulateHidden = () => {
```

Comment:
If the height is also an issue, maybe it would be better to name this method `simulateZeroWidth` and have another called `simulateZeroHeight`,

Reply:
Finally the height has no impact